tests/v2/actions/workloads/deployment/verify.go

```diff
@@ -34,3 +37,52 @@ func VerifyDeployment(steveClient *steveV1.Client, deployment *steveV1.SteveAPIO
 
 	return err
 }
+
+// VerifyRolloutHistoryDeployment is a helper to validate rollout history deployment
+func VerifyRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, expectedRevision string) error {
+	wranglerContext, err := client.WranglerContext.DownStreamClusterWranglerContext(clusterID)
+	if err != nil {
+		return err
+	}
+
+	latestDeployment, err := wranglerContext.Apps.Deployment().Get(namespaceName, deploymentName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if latestDeployment.ObjectMeta.Annotations == nil {
+		return errors.New("revision empty")
+	}
+
+	revision := latestDeployment.ObjectMeta.Annotations[revisionAnnotation]
+
+	if revision == expectedRevision {
+		return nil
+	}
+
+	return errors.New("revision not found")
+}
+
+// VerifyOrchestrationStatus is a helper to verify the orchestration status
+func VerifyOrchestrationStatus(client *rancher.Client, clusterID, namespaceName string, deploymentName string, isPaused bool) error {
+	wranglerContext, err := client.WranglerContext.DownStreamClusterWranglerContext(clusterID)
+	if err != nil {
+		return err
+	}
+
+	latestDeployment, err := wranglerContext.Apps.Deployment().Get(namespaceName, deploymentName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if latestDeployment.Spec.Paused == isPaused {
+		return nil
+	}
+
+	if isPaused {
+		return errors.New("the orchestration is paused")
+	}
+
+	return errors.New("the orchestration is active")
```

Comment:
also i think applying the idea I gave from the other function would make sense for readability; we usually put the 'passing' / non-error use case as the last one, and if statements are where we throw errors

```

can you switch this to the inverse case just so that the structure is in line with other verifies? i.e.
```